client/src/cordova/apple/README.md

```diff
@@ -167,6 +167,8 @@ It's safe to unregister all the Outline VPN Extensions, since the system will lo
 pluginkit -r $APP_EXTENSION_PATH
 ```
 
+Make sure that you list the registered plugins again after unregistering them, since they may fallback to other versions of it, which you may also need to unregister.
```

Comment:
New finding on debugging. We need to keep unregistering and checking until they are all gone.